packages/components/src/SubHeaderBar/InputTitleSubHeader/InputTitleSubHeader.test.js

```diff
@@ -69,6 +66,12 @@ describe('TitleSubHeader', () => {
 		expect(wrapper.getNode().props.className).toEqual('tc-subheader-details-text-subtitle');
 		expect(wrapper.getNode().props.children).toEqual('mySubTitle');
 	});
+	it('should render with with edit pencil', () => {
+		const wrapper = shallow(<TitleSubHeader {...defaultProps} subTitle="mySubTitle" edit />).find(
+			'small',
+		);
+		expect(wrapper.find(Action)).toHaveLength(0);
```

Comment:
I don't understand this test. You render a subheader, you get the subtitle part and check that there are no action on it.
* what's the purpose as there are no action possible in this part
* the test is named `should render with with edit pencil`

Reply:
Bad copy paste... sorry